internal/handlers/common/error.go

```diff
@@ -41,7 +41,7 @@ const (
 	ErrNamespaceExists   = ErrorCode(48)    // NamespaceExists
 	ErrCommandNotFound   = ErrorCode(59)    // CommandNotFound
 	ErrNotImplemented    = ErrorCode(238)   // NotImplemented
-	ErrSortBadValue      = ErrorCode(15974) // Location15974
```

Comment:
hm, it means that some test cases are missing.  Would you please add them?

Reply:
Could you please point to one?

Reply:
There is one test we can add for this error type. Could you please add this test in `TestQueryBadSortType`?
```go
		"BadSortTypeValue": {
			command: bson.D{
				{"find", collection.Name()},
				{"projection", bson.D{{"value", "some"}}},
				{"sort", bson.D{{"asc", "123"}}},
			},
			err: &mongo.CommandError{
				Code:    15974,
				Name:    "Location15974",
				Message: `Illegal key in $sort specification: asc: "123"`,
			},
		},
```

Reply:
@w84thesun good example, thank you.

Reply:
Thank you both for comments. Error case has been added. Error Code has been returned back, but slightly refactored.